independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/buildtool/maven/base/pom.tpl.qute.xml

```diff
@@ -1,6 +1,17 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    {#if project.parent.artifact-id}
```

Comment:
depending on #18227 this will fail if user users older cli targeting new quarkus versions so should do a `?: false` in case parent is not availlable.

Reply:
we are not using strict rendering in code.quarkus if I remember right

Reply:
https://github.com/quarkusio/quarkus/pull/18227/files#diff-78f505a9d2c1e8b7b55a9bd2f0971923696db767bd92070eb719f8ef9ff54cfcR57

Reply:
@ia3andy any reason why we wouldn't want to run with strict rendering at least during tests so our templates explicitly handles these cases ? as otherwise we might not even notice actual errors due to spelling or accidentally false expressions ?

Reply:
i.e. in the PR about qute strict rendering i dont see us turning strict rendering off ? or are we using separate configured qute ?